contracts/LinniaPermissions.sol

```diff
@@ -59,16 +68,101 @@ contract LinniaPermissions is Ownable, Pausable, Destructible {
         return permissions[dataHash][viewer].canAccess;
     }
 
+    /// Add a delegate for a user's permissions
+    /// @param delegate the address of the delegate being added by user
+    function addDelegate(address delegate)
+    onlyUser
```

Comment:
It may be best to make this onlyOwner. So only the admin can add delegates.

Reply:
I think a user should be able to add their own delegates and also has the authority to only allow delegates they approve.

Reply:
I second this. The user needs to be able to set the ddex as a delegate for the registration flow to work

Reply:
Ok. Let go with that.  Let try to make it easy for end users to see the list of their delegates.